examples/random_module.py

```diff
@@ -6,6 +6,7 @@
 bad = random.randrange()
 bad = random.randint()
 bad = random.choice()
+bad = random.choices()
```

Comment:
You've introduced a new functional test by including this line.  As a result the tests will fail, because Bandit will flag this line as a vulnerability.  You'll want to update test_functional.test_random_module()

Reply:
I see.  Okay, thank you.